bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "adapt-contrib-assessment",
-  "version": "3.0.1",
+  "version": "3.0.2",
```

Comment:
I think this should be 3.1.0 as you're effectively adding a new feature

Reply:
Good point.  I've updated.